src/main/java/net/rptools/maptool/client/functions/TokenLightFunctions.java

```diff
@@ -322,6 +322,10 @@ private static LightSource createUniqueLightSource(
             : LightSource.Type.NORMAL;
     final boolean scaleWithToken =
         lightSourceDef.has("scale") ? lightSourceDef.get("scale").getAsBoolean() : false;
+    final boolean ignoreVBL =
+        lightSourceDef.has("ignores-vbl")
+            ? lightSourceDef.get("ignores-vbl").getAsBoolean()
```

Comment:
We should be consistent with the campaign properties syntax about whether to use `ignore-vbl` or `ignores-vbl` otherwise it will lead to confusion. I slightly prefer `ignores-vbl`, but doesn't really matter.

The flag should also be added to the result in `lightSourceToJson()`, and should be added to the result of `getInfo("campaign")` (impl is in `getInfoFunctions()`).